mautic/mautic
# Add notification settings for webhook and campaign unpublish

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for all features, enhancements and bug fixes (until 3.3.0 is released) 
| Bug fix?                               | no
| New feature?                           | yes
| Deprecations?                          | 
| BC breaks?                             | 
| Automated tests included?              | yes
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 




#### Description:
As a user  if I am not the owner of certain resources within the instance I will not receive notifications about actions such as campaigns unpublishing. I want to be able to configure one to many email addresses to receive key notifications for my instance



#### Steps to test this PR:
Require run `doctrine:migrations:execute 20210217115150`

1. Load up [this PR](https://mautibox.com)
2.Switch to Configuration > Notification Settings
3. Yes/No button should be available for Send notification to author
4. If select No then new field should be display for Email addresses to receive notification here user can add multiple comma separated email addresses.
5. Above 3 step should be performed for both section (Campaign and Webhook)
6. Send mail when campaign or webhook unpublished due to failure should work as per above configuration.




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10299"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
